invoice.go

```diff
@@ -68,6 +79,12 @@ type InvoiceInput struct {
 	Metadata      []InvoiceMetadataInput `json:"metadata,omitempty"`
 }
 
+type OneOffInput struct {
```

Comment:
This the namespace is global for all clients:
```suggestion
type InvoiceOneOffInput struct {
```